Gemfile

```diff
@@ -6,6 +6,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem "carrierwave"
```

Comment:
I'd probably shove this gem further down (the gemfile acts as a sort of de-facto documentation, so we want to roughly have the most important stuff first) and comment it to say what it's used for, like the other gems here are.

Usually our code tells us what it's doing — but dependencies have weird names so it's good to leave a comment here just to explain what it does and why we're using it!